lib/blocks/empty-conf.js

```diff
@@ -0,0 +1,21 @@
+module.exports = blockConfig => (processEnv, argv) => config => {
```

Comment:
Rather than having this as the 1st block, which I think can cause issues. Why don't you make it as `buildEmptyConfig` method. And after use all other block in a compose chain, with this method return as the main argument of compose ? Wouldn't it be clearer ?

Reply:
I think I see what you mean. Will give it a try